pom.xml

```diff
@@ -41,6 +41,7 @@ limitations under the License.
         <module>flink-cdc-runtime</module>
         <module>flink-cdc-e2e-tests</module>
         <module>flink-cdc-pipeline-udf-examples</module>
+        <module>flink-cdc-openai-udf</module>
```

Comment:
Maybe putting it as a part of `SystemFunctionUtils` is enough? Creating a new top-level package seems too much.